indy_node/server/pool_req_handler.py

```diff
@@ -34,6 +38,13 @@ def authErrorWhileAddingNode(self, request):
         if error:
             return error
 
+        dest = operation.get(TARGET_NYM)
```

Comment:
Should we check it when we edit NODE txn (changing TARGET_NYM)?

Reply:
Will be extended for verkey changing